node/src/components/consensus/era_supervisor.rs

```diff
@@ -584,6 +584,17 @@ where
         }
 
         self.is_initialized = true;
+        let active_era_outcomes = self.active_eras[&self.current_era]
+            .consensus
+            .handle_is_current();
+        match result_map.entry(self.current_era) {
+            Entry::Occupied(mut current) => {
+                let _ = current.get_mut().extend(active_era_outcomes);
+            }
+            Entry::Vacant(vacant) => {
+                let _ = vacant.insert(active_era_outcomes);
+            }
+        };
```

Comment:
Would this work, too?
Also, the entry is probably guaranteed to exist?
(Not important now, but maybe when we port this to `dev`.)
```suggestion
        result_map.entry(self.current_era).or_default().extend(active_era_outcomes);
```

Reply:
it does! Thanks!